src/main/java/uk/gov/hmcts/cmc/claimstore/services/pilotcourt/PilotCourtCSVHeader.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.hmcts.cmc.claimstore.services.pilotcourt;
+
+public enum PilotCourtCSVHeader {
+    //TODO Comment on Order is important
```

Comment:
What does this mean?

Reply:
Note to myself to add a comment on the importance of the ordering of this enum. Which i then forgot to do. Will update.